src/Plugin/RulesExpression/ActionSet.php

```diff
@@ -27,7 +27,17 @@ protected function allowsMetadataAssertions() {
    * {@inheritdoc}
    */
   public function executeWithState(ExecutionStateInterface $state) {
+    // Get hold of actions.
+    // @todo See if we can add getExpressions method of ExpressionContainerBase.
+    $actions = [];
```

Comment:
Is it really necessary to put in an extra loop just for the copying to a local variable? Can't you just
`$actions = $this->actions;` or even just sort the class property directly? (Ditto for the other applicable classes.)